app/src/test/java/org/simple/clinic/editpatient/EditPatientViewRendererTest.kt

```diff
@@ -47,4 +52,27 @@ class EditPatientViewRendererTest {
     verify(ui).showProgress()
     verifyNoMoreInteractions(ui)
   }
+
+  @Test
+  fun `when input fields are loaded, then setup the ui`() {
+    // given
+    val date = LocalDate.parse("2018-01-01")
+    val userClock = TestUserClock(date)
+    val inputFieldsList = BangladeshInputFieldsProvider(dateOfBirthFormat, LocalDate.now(userClock))
+        .provide()
+    val inputFields = InputFields(inputFieldsList)
+
+    val inputFieldsLoadedModel = model
+        .updateDateOfBirth("2018-03-09")
```

Comment:
Ah, right. We could set this in the default patient profile when creating the model. I must have gone with this, since I forgot to do it when creating the patient profile. Will update it.

Reply:
Was this change actually made? I still don't see the need to manually set a DOB in the model here 🤔